tutorials/btp-integration-suite-execute-batch-payload/btp-integration-suite-execute-batch-payload.md

```diff
@@ -0,0 +1,377 @@
+    		<dbTableName action="INSERT">
+    			<table>samplejdbctest</table>
+    			<access>
+    				<emp_id>4</emp_id>
+    				<emp_name hasQuot="Yes"> g28</emp_name>
+
+    			</access>
+    		</dbTableName>
+    	</insert_statement2>
+    </root>
+    ```
+      <!-- border -->![Non-Atomic Payload](Payload-Non-Atomic.png)
+
+      Check the MPL too.
+
+    <!-- border -->![Payload MPL](Payload-Processed-Non-Atomic.png)
+
+3. Let's confirm, whether the unique record (with `emp_id` 4) is inserted in the database or not. To do this, uncheck the **Batch Mode** checkbox and **Deploy** your integration flow.
+
+4. **Send** the following query from Postman:
+    `select * from samplejdbctest`  
+
+      <!-- border -->![Non-Atomic Batch Mode Verification](Verify-Non-Atomic.png)
+
+So, it is evident that using Non-Atomic mode, the new record is getting inserted and the duplicate record fails. So, the whole payload doesn't fail unlike Atomic Batch operation.
+
+>The function/behaviour of Non-Atomic Batch Operation may depend upon the choice of database.
+
+### Send Payload using Groovy Script
+
+You can also upload the payload using the **Groovy Script**. In this case you must use the prepared statements.
+
+
+1. Uncheck **Batch Mode** in the integration flow.
+<!-- border -->![Batch Mode Off](Batch-Off.png)
+Let's insert a new database table. We have already seen creating sample database via **Content Modifier**. Now we will do it directly from Postman. **Send** this query from Postman.
+
+    `create table testjdbcsample (emp_id integer primary key, emp_name varchar (255), join_date varchar(255), a1 integer, a2 integer, email varchar(255));`
+
+      <!-- border -->![Create database from Postman](Database-Postman.png)
+
+2. In the integration flow, insert the **Groovy script** to pass the payload and click **Select**.
+
+    <!-- border -->![Insert Groovy Script](Groovy-Script.png)
+
+    ```
+    import com.sap.gateway.ip.core.customdev.util.Message;
+    import java.util.HashMap;
+    import java.util.Arrays;
+
+    //Insert script
+    def Message processData(Message message) {
+
+        //Headers
+        List paramList = new ArrayList();
+        paramList.add(Arrays.asList(108, 'test', 2021-01-01,1,1,'test@gmail.com'));
+        paramList.add(Arrays.asList(107, 'test', 2021-01-01,1,1,'test@gmail.com'));
+        paramList.add(Arrays.asList(111, 'test', 2021-01-01,1,1,'test@gmail.com'));
+        paramList.add(Arrays.asList(105, 'test', 2021-01-01,1,1,'test@gmail.com'));
+        message.setHeader("CamelSqlParameters",paramList);
+
+        //Body
+        message.setBody("INSERT INTO testjdbcsample (emp_id,emp_name,join_date,a1,a2,email) VALUES(?,?,?,?,?,?)");
+        return message;
+    }
+    ```
+3. Select the **Batch Mode** checkbox and click **Deploy**.
+
+4. Run a blank query in Postman to send the payload which we have entered via Groovy Script and establish a connection between Postman and your integration flow.
```

Comment:
Don't say "we" when addressing the author. Say "you" instead!
Missing comma before "which"

Reply:
done and reframed